service/db/src/main/java/gov/va/vro/service/db/ClaimMetricsServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package gov.va.vro.service.db;
+
+import gov.va.vro.persistence.repository.ClaimRepository;
+import gov.va.vro.service.spi.services.ClaimMetricsService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class ClaimMetricsServiceImpl implements ClaimMetricsService {
+
+  private final ClaimRepository claimRepository;
+
+  @Override
+  public Integer claimMetrics() {
```

Comment:
I would not return Integer but an POJO. We will improve these metrics soon enough.